src/components/ErrorModal/ErrorModal.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Button,
+  Col,
+  Modal,
+  Row,
+} from '@folio/stripes/components';
+
+import { FormattedMessage } from 'react-intl';
+
+class ErrorModal extends React.Component {
+  static propTypes = {
+    open: PropTypes.bool,
+    onClose: PropTypes.func,
+    message: PropTypes.string,
+  };
+
+  render() {
+    const {
+      open,
+      message,
+      onClose,
+    } = this.props;
+
+    return (
+      <Modal
```

Comment:
@aditya-matukumalli could we use `footer` prop here together with `<ModalFooter>`? I think it would cleanup part of the modal. Here is a little snippet: 

https://github.com/folio-org/ui-checkout/pull/236/files#diff-a5f5e17a9a0a4d91adeac5f9c56bacafR11

Reply:
@mkuklis Thanks. Looks much cleaner